etc/ctags.readme.txt

```diff
@@ -1,9 +1,44 @@
-Copy the file ctags_for_gap to ~/.ctags and then run
-  ctags --sort=no lib/*.g lib/*.gi lib/*.gd src/*
-in the GAP installation directory. Start your editor
-in that directory and then do (for vim e.g.):
+This file describes how to set up Ctags to work with GAP source files.
+Ctags is a tool that can be used to generate an index file of names found in
+source files. This index file can then be used by editors to jump to the
+definition of any indexed name.
+
+Note that if you have Exuberant Ctags installed (if in doubt, run `ctags --version`)
+and only want to generate tags for the gap-system/gap repository, then you do
+not need set up anything. The symlink `.ctags` in the root folder already
+tells ctags all it needs to know, and you can create and update the tags by
+running:
+  make tags
+If you want to use ctags in another GAP project or have Universal Ctags
+installed, read on.
+
+Run
+  ctags --version
+to see whether you are running Exuberant Ctags or Universal Ctags.
+Append the content of the file ctags_for_gap to the file
+  ~/.ctags
+if you are running Exuberant Ctags. If you are running Universal Ctags then
```

Comment:
Make the two cases clearer here. Maybe use paragraphs?

Reply:
Bonus idea: convert the file to markdown and call it `README.ctags.md`; then use four space indent and surrounding empty lines to turn this into code blocks.

But of course this goes way beyond the original scope of this PR. so please do not feel compelled to do this unless you really want to.